base/views.jl

```diff
@@ -113,11 +113,13 @@ macro view(ex)
     if Meta.isexpr(ex, :ref)
         ex = replace_ref_begin_end!(ex)
         if Meta.isexpr(ex, :ref)
-            ex = Expr(:call, view, ex.args...)
+            ex = Expr(:call, Base.view, ex.args...)
```

Comment:
This may be intentional — and I wouldn't be surprised if folks somewhere are depending upon this.

Reply:
It actually doesn't make a difference because it's putting in the `view` function as a value. You may probably either want `:view`, or :(Base.view) or  `GlobalRef(Base, :view)`.

Reply:
Is there a problem with leaving it as a value?

Reply:
The AST probably serializes better without the explicit object, but it doesn't really matter.

Reply:
Oh duh, right.  I really just figured that SGJ wouldn't get this wrong.  Hah!